src/main/java/uk/nhs/hee/trainee/details/dto/PlacementDto.java

```diff
@@ -38,9 +40,9 @@ public class PlacementDto implements SignedDto {
   private String tisId;
   private LocalDate startDate;
   private LocalDate endDate;
-  private String site;
-  private String siteLocation;
-  private String siteKnownAs;
+  @JsonUnwrapped
```

Comment:
I like this - keeps things simple for the main site.
So the placement json will look something like `{... "name": "site name", "knownAs": "site know as", "siteLocation": "site location", "otherSites": [ { "name": "other site name 1", "knownAs": "other site know as", "siteLocation": "other site location" }, { "name": "other site name 2", "knownAs": "other site know as", "siteLocation": "other site location" } ] ... }` right?

Reply:
An alternative would be something like below, but it would be a breaking change (and we'd have to make up a site "type" for the main site since the only type available in TIS is "OTHER".

```json
{
  "tisId": "1538788",
  ...
  "sites": [
      {
          "name": "Norwood Medical Centre",
          "knownAs": "Norwood Medical Centre (A82008)",
          "location": "Norwood Medical Centre 99 Abbey Road Barrow-In-Furness Cumbria",
          "type": "MAIN"
      },
      {
          "name": "The Royal Liverpool University Hospital",
          "knownAs": "The Royal Liverpool University Hospital (RQ617)",
          "location": "Prescot Street  LIVERPOOL",
          "type": "OTHER"
      },
      {
          "name": "Norwood Medical Centre",
          "knownAs": "Norwood Medical Centre (A82008)",
          "location": "Norwood Medical Centre 99 Abbey Road Barrow-In-Furness Cumbria",
          "type": "OTHER"
      }
  ],
  ...
}
```
(Ignore the same site being both `MAIN` and `OTHER`, that's just down to my limited test data.

Reply:
I think consistency is ok here. I'm assuming the specialty work will simply add another field for subSpecialty, not restructure the object to have a `{... "specialties": [ { "name": "some specialty", "type": "PRIMARY" }, { "name": "some other specialty", "type": "SUB_SPECIALTY" } ] ...}` collection!